src/components/ClauseComponent.js

```diff
@@ -111,11 +143,36 @@ function ClauseComponent(props) {
         ]
       };
       const newInlineSlate = Inline.fromJSON(newInlineJSON);
-
       props.editor.replaceNodeByKey(selectionNodeKey, newInlineSlate);
     }
   };
 
+  const addList = (listLastkey,parentKey) =>{
+    const listLastNode = props.editor.value.document.getNode(listLastkey);
+    const parentNode = props.editor.value.document.getNode(parentKey);
+
+    const newBlockJSON = regenerateKey(listLastNode);
+    const newBlockSlate = Block.fromJSON(newBlockJSON);
+    
+    let newNode = parentNode.nodes.asMutable();
+    
+    newNode =newNode.push(newBlockSlate);
```

Comment:
Might not be possible, but could this be one line? 

```js
const newNode = parentNode.nodes.asMutable().push(newBlockSlate);
```

Reply:
Yeah, it could, will change it